config/locales/publishers.yml

```diff
@@ -94,6 +94,7 @@ en:
         page_title: Set up your Teaching Vacancies account
       form:
         schools_in: Schools in %{organisation} local authority
```

Comment:
Wrong PR, but: would it be simpler to update the school import task so that local authorities have 'local authority' added in the name attribute? That way we don't have to duplicate translations or adapt translations depending on what type of organisation they are being used for. E.g. will 'publisher preferences' always be used by local authorities?

Reply:
I wouldn't alter what we get from GIAS